### PR TITLE
Mob Corpses Fix

### PR DESCRIPTION
**Pull request by clusterfack**
Fixes mob corpses processing life() needlessly by removing them from the mob lists
Removes an almost identical second method of creating corpses, also fixes MOB RUSSIAN CORPSE DOES NOT INHERIT NEW error at start.

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/syndicate.dm [hunk 1]
```diff
@@ -37,12 +37,12 @@
 /mob/living/simple_animal/hostile/syndicate/Die()
 	..()
 	if(corpse)
-		new corpse (src.loc)
+		corpse.createCorpse()
```
Comment (ComicIronic) [inline code/modules/mob/living/simple_animal/hostile/syndicate.dm]: Does this work with just a type?

Comment (clusterfack) [inline code/modules/mob/living/simple_animal/hostile/syndicate.dm]: My code just compiled but yes it should, the only problem that can occur will come from the loc thats used.

Comment (ComicIronic) [inline code/modules/mob/living/simple_animal/hostile/syndicate.dm]: The issue I have with it is that createCorpse references src, which is an issue if you don't actually new the landmark at any point.

Comment (clusterfack) [inline code/modules/mob/living/simple_animal/hostile/syndicate.dm]: When you have a variable of a type like this it should read the data from the type path of the variable, I believe I've made something almost identical to this work with windows in the past which is why I've applied it here. 

But I'll be certain whether or not it works soon.

**Comment by clusterfack** [comment]
Fixes #3674

**Comment by d3athrow** [comment]
Like I said in IRC, they process life 5 times and then stop.

**Comment by clusterfack** [comment]
That's not true

**Comment by d3athrow** [comment]
https://github.com/d3athrow/vgstation13/blob/Bleeding-Edge/code/modules/mob/living/carbon/human/life.dm#L249

**Comment by clusterfack** [comment]
Give me a minute to collect some equally damning evidence

**Comment by clusterfack** [comment]
So yeah you are right that once they die, they take 5 ticks to stop processing life(). 

However the roundstart landmark corpses start out at full health and have to die first, which takes about 130 life() ticks to complete.

So I'm going to take out the two lines that take them out of the mob list, and consider this a refactor of the way landmark corpses work (I made them die instantly now)

**Comment by clusterfack** [comment]
Alright this is completed

**Pull request merged**